roles/sap_anydb_install_oracle/.yamllint.yml

```diff
@@ -20,3 +22,6 @@ rules:
 #  new-lines: {type: unix}
 #  trailing-spaces: disable
   truthy: disable
+  octal-values:
```

Comment:
This values are by default, do we want to keep just to be sure and to be more clear?

Reply:
The only reason is that ansible-lint 24.7.0 demands that these values are specified, otherwise it will complain as I showed in the output in https://github.com/sap-linuxlab/community.sap_install/pull/811. So it seems that there is no alternative to my proposal. But if you can think of one or find one, please let me know so that we can change it. Maybe future ansible-lint versions behave differently - in this case we can remove these lines again.

Reply:
You are right, I forgot that warning. Sorry!